### DISCUSSION TITLE
display of remaining time and percentage calculation

### DISCUSSION THREAD

**Discussion by BUZZ-SEB** [Guilouz/Creality-Helper-Script-Wiki#595] [Issues]
bonjour, 
quand j'envoi un pièce dans orcaslicer avec mon imprimante ender 3 V3 KE, le temps restant et le pourcentage ne s'affiche pas sur le panneau de contrôle ou sur creality cloud. j'ai testé MAINSAIL ou FLUIDD c'est pareil.
avez vous une idée?
merci

Hi, when I send a piece in orcaslicer with my ender 3 V3 KE printer, the remaining time and the percentage are not displayed on the control panel or on creality cloud. I tested MAINSAIL or FLUIDD, it's the same.
do you have an idea?
Thanks

![IMG_0665](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/170262312/db75f754-d506-4e49-8f85-bb3487c3a26d)
![IMG_0666](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/170262312/ea34583e-08ea-4b27-841f-cfdb131ea40f)

**Comment by Guilouz** [comment]
Depending what slicer you use?

**Comment by BUZZ-SEB** [comment]
Bonjour, j’utilise ORCASLICER. Cordialement


Hi, I’use ORCASLICER. Have a nice day, best regards.
